tests/integration/api_admin_org_test.go

```diff
@@ -83,6 +82,6 @@ func TestAPIAdminOrgCreateNotAdmin(t *testing.T) {
 		Location:    "Shanghai",
 		Visibility:  "public",
 	}
-	req := NewRequestWithJSON(t, "POST", "/api/v1/admin/users/user2/orgs?token="+token, &org)
-	session.MakeRequest(t, req, http.StatusForbidden)
+	req := NewRequestWithJSON(t, "POST", "/api/v1/admin/users/user2/orgs", &org)
+	session.MakeRequest(t, req, http.StatusUnauthorized)
```

Comment:
We should use `MakeRequest` directly. All `/api/v1` routes will not read session.

Reply:
I think it's the correct behavior here, and other tests were using `MakeRequest` before this PR as well. I thought `MakeRequest` is used to check for expected http status, rather than letting the API handler to use the session. So here the absence of `token` means it will result in `http.StatusUnauthorized` response.

Reply:
I sent #21718 to make it more clear.